tests/hotplug/memory.go

```diff
@@ -158,7 +159,10 @@ var _ = Describe("[sig-compute][Serial]Memory Hotplug", decorators.SigCompute, d
 			Expect(compute).NotTo(BeNil(), "failed to find compute container")
 			reqMemory = compute.Resources.Requests.Memory().Value()
 			Expect(reqMemory).To(BeNumerically(">=", newGuestMemory.Value()))
-		})
+		},
+			Entry("with a common VM"),
+			Entry("with 2Mi pagesize hugepages VM", decorators.RequiresHugepages2Mi, libvmi.WithHugepages("2Mi")),
```

Comment:
This option is passed but there is no argument in the table's function

Reply:
darn, I'll add it.

Reply:
Fixed it.

Reply:
:man_facepalming: <- on me